test/services/RosmPatternServiceSpec.scala

```diff
@@ -32,16 +32,19 @@ class RosmPatternServiceSpec extends BaseSpec {
 
   private val SUT = new RosmPatternServiceImpl(mockDesService, mockTaxEnrolmentsService)
 
-  ".completeRosmTransaction" should {
+  ".completeRosmTransaction 4MLD" should {
+
+    val taxable: Boolean = false
+    val trn = "XTRN1234567"
```

Comment:
Are we missing specs for 5MLD?

Reply:
tests updated to pass taxable flag and trn